src/components/Filter/OptionDisplay.tsx

```diff
@@ -89,7 +89,7 @@ export const OptionDisplay = ({
     return (
       <button
         className={classes.option}
-        key={option.value}
+        key={option.label}
```

Comment:
because resource register returns both Arbeids and velferdsetaten and Nav (Arbeids og velferdsetaten)

Reply:
Does that mean we will now have both Arbeids and velferdsetaten and Nav (Arbeids og velferdsetaten) in our display?
That sounds suboptimal...

Reply:
I don't see any other solution than checking if 2 resourceOwners have the same org code and then only return one of them (We could then add a rule for which of them to return in those scenarios but I don't know what rule we should use for which of them to use, maybe we should huddle about it tomorrow?)

Reply:
Yeah, no I don't think we should try and fix every issue that isn't our fault with special case code.
That being said though,, I thought the way the code was written in the BFF was that we filtered out the orgs from the list of resources, based on the orgcode or orgnumber? So shouldn't that have meant that we wouldn't get duplicates like these?
(I haven't actually cheched right now though, so I might be wrong)